influence_benchmark/RL/run_EI_iteration.py

```diff
@@ -14,6 +15,8 @@
 )
 from influence_benchmark.utils.utils import set_all_seeds
 
+SFT_TRAINING_PATH = os.path.abspath(__file__)
+
```

Comment:
Could be nicer to use path like in root?

Reply:
The thing I thought was nice about it is that this is immune to refactorings of the file name and location. What are the advantages of having it in `root.py`?

Reply:
I meant use the path library haha